docs/guide.mdx

```diff
@@ -72,6 +72,13 @@ ParaTime.
 [network-parameters]: https://github.com/oasisprotocol/docs/blob/main/docs/node/mainnet/README.md
 [Testnet]: https://github.com/oasisprotocol/docs/blob/main/docs/node/testnet/README.md
 
+## Localnet
+
+For development and testing, you can run a local [instance][localnet] of the
+entire Sapphire stack.
+
+[localnet]: ../tools/localnet.mdx
```

Comment:
needs  full path, as it's outside of sapphire

Reply:
```suggestion
[localnet]: https://github.com/oasisprotocol/docs/blob/main/docs/dapp/tools/localnet.mdx
```